src/api/spec/features/webui/packages_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'browser_helper'
 require 'webmock/rspec'
 
+# CONFIG['global_write_through'] = true
```

Comment:
We don't that anymore, please remove it :+1:

Reply:
why we don't need it anymore?

Reply:
We do it now in the `spec/support/vcr.rb` file instead of spread in the spec files.